src/algebra/big_operators/fin.lean

```diff
@@ -5,6 +5,7 @@ Authors: Anne Baanen
 -/
 import algebra.big_operators.basic
 import data.fintype.fin
+import data.fintype.card
 /-!
 # Big operators and `fin`
 
```

Comment:
Can you add a brief list of the main results to the end of docstring? I would say `fin.prod_univ_succ` and `fin.prod_const` are the most useful ones (along with their sum variants)

Reply:
Ok.